aapcs64/aapcs64.rst

```diff
@@ -898,10 +898,44 @@ registers, or if it is a function that returns results in such registers,
 it must ensure that p4-p15 are preserved across the call. In other cases
 it need not preserve any scalable predicate register contents.
 
-Processes, Memory and the Stack
--------------------------------
+Threads and processes
+---------------------
+
+The AAPCS64 applies to a single _`thread` of execution.  Each thread is in
+turn part of a _`process`.  A process might contain one thread or several
+threads.
+
+The exact definition of “thread” and “process” depends on the platform.
+If the platform is a traditional multi-threaded operating system,
+“thread” and “process” generally have their usual meaning for that
+operating system. If the platform supports multiple processes but has
+no separate concept of threads, “thread” is synonymous with “process”.
+If a platform has no concurrency or preemption then it will generally
+have a single “thread” and “process” that executes all instructions.
```

Comment:
Avoid using quotation marks for emphasis - see Writing Guide https://confluence.arm.com/display/TCOM/Avoid+formatting+for+emphasis

Reply:
I wasn't really intending to use quotes for emphasis here.  This paragraph is trying to define what the ABI means by the terms “thread” and “process” (sometimes in circumstances where the concept of thread and process might seem unnatural).

Reply:
In this case - I would just use the quotes in the first instance of the term in in the paragraph.

Reply:
:)

Reply:
I'd rather keep the paragraph here.  But I've tried to reword it to avoid excessive quoting.